src/control/cmd/daos_server/storage_utils.go

```diff
@@ -175,12 +196,26 @@ func getSockFromCfg(log logging.Logger, cfg *config.Server, affSrc config.Engine
 }
 
 func (cmd *scmCmd) init(ctx context.Context) error {
+	if cmd.ctlSvc != nil {
+		// Allow parseOpts() unit tests to skip contents of init().
```

Comment:
Is it mandatory to have this mocking inside the code ?

Reply:
as above